src/components/DropDown/index.tsx

```diff
@@ -0,0 +1,71 @@
+import cn from 'classnames';
+import React, { ChangeEvent } from 'react';
+import { Person } from '../../types/Person';
```

Comment:
it is better to write the full name of the library that is used as a variable (instead of cn - classnames)

Reply:
It is actually okay to use cn instead of classnames, because a lot of developers do that (including misha), so that is a kind of a thing that is agreed on a project
+ it saves up "space" in the line of code 🤗